enterprise/backend/src/metabase_enterprise/audit_db.clj

```diff
@@ -84,18 +92,18 @@
   "Default Dashboard Overview (this is a dashboard) entity id."
   "bJEYb0o5CXlfWFcIztDwJ")
 
-(def ^{:arglists '([audit-installed? model entity-id])
+(def ^{:arglists '([checksum model entity-id])
        :private  true} memoized-select-audit-entity*
   (mdb/memoize-for-application-db
-   (fn [audit-installed? model entity-id]
-     (when audit-installed?
+   (fn [checksum model entity-id]
+     (when checksum
```

Comment:
I don't think this fixes the issue. `checksum` is never falsy and has a default of 0. So currently this would execute in states. Should this be `(when (not (zero? checksum)) ...)`? Or maybe the default for `last-analytics-checksum` should be nil, rather than having 0 be a special 'unset' value?

Reply:
Good catch. Changed the default to `nil`